lib/rules/no-only.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @fileoverview Don't allow test.only to be added to the repository
+ * @author Ben Monro
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "Don't allow `test.only` to be added to the repository",
+            category: "Mistake Prevention",
+            recommended: true
+        },
+        fixable: null,
+        schema: []
+    },
+
+    create: function (context) {
+
+        return {
+
+            "MemberExpression[object.name=/fixture|test/][property.name='only']"(node) {
```

Comment:
The `.skip` and `.only` hooks can be located not only after the `fixture` or `test` directives.
```js
fixture `Fixture`
	.page('https://example.com')
	.only;

test('test', async () => {});
```

Reply:
I will add more tests for these but I'm pretty sure they would work